index.js

```diff
@@ -82,36 +82,36 @@ var req = https.request(options, function(res) {
           if(!data.answers.length || data.answers.length==0){
             data.answers.push(defaultAnswer);
           }
+          var actions=data.answers[0].actions;
```

Comment:
I think a more descriptive name would be better. Also, put a space?

Like this : 

```javascript
var answerActions = data.answers[0].actions;
```